File: TriviaClient/Responses/GetStatsResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TriviaClient.Responses
{
    class GetStatsResponse
    {
        float _winRate;
        float _successRate;
        float _stupidityRate;
        float _averageTime;

        public GetStatsResponse(float winRate, float successRate, float stupidityRate, float averageTime)
        {
            _winRate = winRate;
            _successRate = successRate;
            _stupidityRate = stupidityRate;
            _averageTime = averageTime;
        }

        public float GetWinRate()
        {
            return _winRate;
        }

        public float GetSuccessRate()
        {
            return _successRate;
        }

        public float GetStupidityRate()
        {
            return _stupidityRate;
        }

        public float GetAverageTime()
        {
            return _averageTime;
        }
    }
}

File: Trivia/GetStatsResponse.cpp
#include "GetStatsResponse.h"

GetStatsResponse::GetStatsResponse(float winRate, float successRate, float stuipidityRate, float averageTime) : _winRate(winRate), _successRate(successRate), _stuipidityRate(stuipidityRate), _averageTime(averageTime)
{
}

float GetStatsResponse::getWinRate() const
{
	return _winRate;
}

float GetStatsResponse::getSuccessRate() const
{
	return _successRate;
}

float GetStatsResponse::getStuipidityRate() const
{
	return _stuipidityRate;
}

float GetStatsResponse::getAverageTime() const
{
	return _averageTime;
}
